plugins/Rockets/RocketsPlugin.cpp

```diff
@@ -657,6 +660,9 @@ class RocketsPlugin::Impl : public ActionInterface
                 else
                     throttle.second(delayedNotify, delayedNotify, throttleTime);
             });
+
+        _unsetObjectCallbacks.push_back(
+            [&obj] { obj.onModified(BaseObject::ModifiedCallback()); });
```

Comment:
obj has a base class, but I don't see how this would help or how the lambda is confusing/harming here. And I wrote _unsetObjectCallbacks, as this what I exactly I intended and you also acknowledged :)